packages/gatsby-theme-newrelic/src/components/Banner.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const Banner = ({ children, color }) => {
+  const [visible, updateVisible] = useState(true);
```

Comment:
Should we consider persisting this state between page refreshes? Otherwise the user will need to dismiss the banner every time they load the site which could be annoying.

If we go that route, that likely means we need to think about creating some kind of unique ID for the banner so that dismissing the banner doesn't mean the user misses out on new announcements.

Reply:
Great idea!